test/Client/Endpoint/DataRetentionEndpointTest.php

```
@@ -0,0 +1,155 @@
+        $this->assertInstanceOf(\CedricZiel\MattermostPhp\Client\Model\GetDataRetentionPoliciesCountResponse::class, $result);
+    }
+
+    #[Test]
+    public function getDataRetentionPoliciesBuildsCorrectRequest(): void
+    {
+        $this->mockJsonResponse(200, [['status' => 'ok']]);
+
+        $page = 1;
+        $per_page = 1;
+
+        $result = $this->endpoint->getDataRetentionPolicies($page, $per_page);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)